graphistry/embed_utils.py

```diff
@@ -442,7 +445,10 @@ def predict_link(
         score = self._score(
             torch.from_numpy(test_df.to_numpy().astype(np.float32)).to(dtype=torch.long)
         )
-        result_df = test_df.loc[pd.Series(score.detach().numpy()) >= threshold]  # type: ignore
+        if anomalous:
+            result_df = test_df.loc[pd.Series(score.detach().numpy()) <= threshold]
```

Comment:
confidence <= threshold might not be anomaly, cz we have set very high threshold.

Reply:
`anomalous=True` just switches the comparison so we filter down to low scores.